src/DocTest/Parser.elm

```diff
@@ -96,26 +95,16 @@ commentEnd str =
 
 replacePrefix : String -> String
 replacePrefix str =
-    replace (AtMost 1) (regex "\\s{4}[>|=|\\.]{3}\\s") (\_ -> "") str
-
-
-isDocTestPrefix : String -> Bool
-isDocTestPrefix str =
-    expectationPrefix str
-        || assertionPrefix str
-        || continuationPrefix str
-
-
-expectationPrefix : String -> Bool
-expectationPrefix str =
-    contains (regex "\\s{4}===\\s.*") str
+    str
+        |> replace (AtMost 1) (regex "^\\s{4}") (\_ -> "")
+        |> replace (AtMost 1) (regex "^[>|\\.]{3}\\s") (\_ -> "")
```

Comment:
I'm making a blog post about regexes and how to avoid them breaking things actually :sweat_smile: my recommendation is to pull these out to be at the top level (e.g a name by themselves), since that way you will encounter the error when the page loads, rather than when `expecationPrefix` is called